e2e-tests/cypress/support/step_definitions/email_steps.ts

```diff
@@ -51,9 +55,53 @@ Then('the user receives an e-mail containing the {string} link', (linkName: stri
   )
 })
 
+And('the user receives the transaction e-mail about {string} GDD from {string}', (amount: string, senderName:string) => {
```

Comment:
```suggestion
And('the user received the transaction e-mail about {string} GDD from {string}', (amount: string, senderName:string) => {
```

Reply:
The time form throughout all the features is present.
Let's stick to it for coherence.